### ISSUE TITLE
Rename "connection" and "space"

### ISSUE THREAD

**Issue by alfred-landrum** [brimdata/brim#1231]
After some fun naming discussions, we've landed on using "workspace" instead of "connection". So, a user accesses a workspace by providing a network endpoint and (in the future) authentication credentials, and the workspace contains resources including data, (and in the future) saved dashboards, queries, alerts, etc.

Regarding data, at one time we thought we'd say "lake" in the app where we currently say "space", but we've since changed our minds a bit on that one, since the idea of potentially having lots of small "lakes" in what can be used as a desktop app feels like a mismatch with how "data lakes" are perceived outside of Brim

A straw man proposal from @mccanne as of March, 2021:

> How about “data pool” instead of space?  “A lake can have any number of pools, where data in each pool is organized by its pool ‘key’.  The pool key is used to scalably organize its data using LSM-like algorithms for efficiently rolling up sorted data into cloud-optimized processing units.”

**Comment by alfred-landrum** [comment]
The connection to workspace change was made in https://github.com/brimsec/brim/pull/1372 .

**Comment by philrz** [comment]
As of Brim commit 09efdcd, it looks like we've accomplished what we described here.

What was once called a "Connection" is indeed now described as a "Workspace".

![](https://user-images.githubusercontent.com/5934157/116895380-0041b280-abe8-11eb-9a52-0ae0fd23f631.png)

![](https://user-images.githubusercontent.com/5934157/116895411-09cb1a80-abe8-11eb-80b1-eb81171e54d8.png)

Also, that which were once "Spaces" are now "Pools".

![](https://user-images.githubusercontent.com/5934157/116895490-1bacbd80-abe8-11eb-979a-3adba23360e5.png)
